modules/gitops-deleting-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-deleting-rolloutmanager-custom-resource_{context}"]
+= Deleting a RolloutManager custom resource
+
+Uninstalling the {gitops-title} Operator does not remove the resources that were created during installation. To uninstall the progressive delivery of deployments by using {gitops-title}, you must explicitly delete the resources by deleting the `RolloutManager` custom resource (CR), and then uninstall the {gitops-title} Operator.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0 or a newer version is installed in your cluster.
+* A `RolloutManager` CR exists in your namespace.
+
+.Procedure 
+
+. Log in to the {product-title} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where `RolloutManager` CR for the progressive delivery is setup from the *Project* drop-down menu.
```

Comment:
A couple of nits:

1. "Create or select the project" -> Is it possible for it to have not been created at this point?
2. The phrasing is confusing here. I'd suggest the following revision based on similar documentation:
```
Click the *Project* drop-down menu and select the project that contains the `RolloutManager` CR.
```